File: Java_training/session1/basic/EvenOrOdd.java
package session1.basic;
import java.util.Scanner;
public class EvenOrOdd {
    static void isEvenOrOdd(int num) {
        if (num % 2 == 0) {
            System.out.println(num + " is an even number");
        } else {
            System.out.println(num + " is on odd number");
        }
    }
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter Number: ");
        int n=sc.nextInt();
        isEvenOrOdd(n);
        sc.close();
    }
}

File: prachiverma_python_training/assignnment_1/python_basic/operators_and_conditionals/even_or_odd.py
""" program to check whether the number is even or odd """

def check_even_odd(num):
    if num % 2 == 0:
        return "Even"
    return "Odd"

number = int(input("Enter a number: "))
print(f"The number {number} is {check_even_odd(number)}.")